src/server.ts

```diff
@@ -1,11 +1,14 @@
-import { Application, Request, Response } from 'express';
+// server.ts
+import { Application, Request, Response, NextFunction } from 'express';
 import dotenv from 'dotenv';
 import cors from 'cors';
 import express from 'express';
 import swaggerUI from 'swagger-ui-express';
 import swaggerJsDoc from 'swagger-jsdoc';
 import options from './docs/swaggerdocs';
 import routes from './routes';
+//import { validateCreateProfile, validateUpdateProfile } from './validations/profileValidation';
+import profileRoutes from './routes/profileRoutes';
```

Comment:
remove this please, because you already have the profile route define in the overall routes that is in `/routes/index.ts`